src/Resources/views/Order/Admin/Refund/Body/_form.html.twig

```diff
@@ -0,0 +1,20 @@
+<form action="{{ path('sylius_refund_refund_units', {'orderNumber': app.request.attributes.get('orderNumber')}) }}" method="post">
+    <input type="hidden" name="_csrf_token" value="{{ csrf_token(app.request.attributes.get('orderNumber')) }}" />
+    <table id="refunds" class="ui compact celled table">
+        <thead>
+        <tr>
+            {{ sylius_template_event('sylius_refund.order.admin.refund.table.header', _context) }}
+        </tr>
+        </thead>
+        <tbody>
+        {{ sylius_template_event('sylius_refund.order.admin.refund.table.row', _context) }}
+        </tbody>
+    </table>
+    {% include '@SyliusRefundPlugin/_paymentMethod.html.twig' %}
```

Comment:
In order not to have to include this template here, you could create a single template event `...form.content` with blocks for table, payment method and footer

Reply:
I've decided not to use `content` field as it's quite generic and could be added to every event. In fact, each block could be treated as some kind of content. Wdyt?

Reply:
It makes sense, however, we've used this keyword in template events in Sylius, but for now, there is no rule or any convention about the naming of events